app/services/google_policy_site.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class GooglePolicySite
+  BASE_URL = URI('https://policies.google.com').freeze
+
+  def self.locale_segment
+    active_locale = I18n.locale
+    active_locale == I18n.default_locale ? '' : "?hl=#{active_locale}"
+  end
+
+  def self.base_url
+    URI.join(BASE_URL, locale_segment).to_s
+  end
+
+  def self.privacy_url
+    URI.join(BASE_URL, "/privacy#{locale_segment}").to_s
```

Comment:
Maybe there's some way we could incorporate [`UriService.add_params`](https://github.com/18F/identity-idp/blob/bc3ec5e9e576005d931fd83bebc70132f4164210/app/services/uri_service.rb#L7-L16) for this, to avoid any potential errors with constructing query parameters?